inst/stan/estimate_truncation.stan

```diff
@@ -13,6 +13,7 @@ parameters {
   real logmean[1];
   real<lower=0> logsd[1];
   real<lower=0> phi;
+  real<lower=0> sigma;
```

Comment:
It would be useful to add a note to the function documentation and potentially a news item.